deploy/kubernetes/base/controller-deployment.yaml

```diff
@@ -37,9 +37,8 @@ spec:
           args:
             - --endpoint=$(CSI_ENDPOINT)
             - --logtostderr
-            - --v=5
-            # Uncomment below line to allow access point root directory to be deleted by controller.
-            #- --delete-access-point-root-dir
+            - --v=2
+            - --delete-access-point-root-dir=false
```

Comment:
Do we want this?

Reply:
make generate-kustomize cleans out comments. It's hard to preserve them. TBH, I think it's better if we try to copy what @AndyXiangLi  did for ebs and document these flags in README than keep comments that won't be noticed https://github.com/kubernetes-sigs/aws-ebs-csi-driver#driver-options

Reply:
I misread, thought you meant just the comment being removed.

about it being false by default:

yes, that's intended, it comes from the helm chart making it false by default.

Reply:
one more comment:

about it being false by default in the helm chart in the first place:

discussion is here: https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/411